React/tic-tac-toe/src/app/screens/Login/validation/index.js

```diff
@@ -0,0 +1,15 @@
+export const required = value =>
```

Comment:
this should be in a utils folde at the same level that app folder is. then, inside.. you should have a validations.js file and this functions insede